### DISCUSSION TITLE
Support for Rust

### DISCUSSION THREAD

**Discussion by ranger-ross** [allure-framework/allure2#2782] [Ideas & Feedback]
Are there any plans to support the Rust programming language?

**Comment by yukinarit** [comment]
Hi! 

I am developing a test framework and runner similar to Postman/Playwright in Rust (the project is currently private). I would like to develop a plugin for Allure. I am wondering if anyone is already working on an adapter. Otherwise, I am considering working on my own.

**Comment by bartoszdabr** [comment]
Is it anywhere on the roadmap? The discussion is dead and I see no statement on it :/ There are some community driven crates already but I haven't found any official statement on that

**Comment by baev** [comment]
Now available https://github.com/allure-framework/allure-rust , ETA next week to be published
